node/pkg/driver/node_utils.go

```diff
@@ -210,31 +228,17 @@ func (n NodeUtils) ParseFCPorts() ([]string, error) {
 	}
 
 	for _, fpath := range fpaths {
-		file, err := os.Open(fpath)
+		fcPort, err = readAfterPrefix(fpath, "0x", device_connectivity.ConnectionTypeFC)
```

Comment:
```suggestion
		fcPort, readErr := readAfterPrefix(fpath, "0x", device_connectivity.ConnectionTypeFC)
```
(without `var fcPort string`)

Reply:
done